### PR TITLE
Dubbo test in PelicanDT!

### PR DESCRIPTION
**Pull request by moyunalibaba**
## What is the purpose of the change

XXXXX

## Brief changelog

XXXXX

## Verifying this change

XXXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

- [x] Make sure there is a [GITHUB_issue](https://github.com/apache/incubator-dubbo/issues) field for the change (usually before you start working on it). Trivial changes like typos do not require a GITHUB issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
- [ ] Format the pull request title like `[Dubbo-XXX] Fix UnknownException when host config not exist #XXX`. Each commit in the pull request should have a meaningful subject line and body.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/apache/incubator-dubbo/tree/master/dubbo-test).
- [ ] Run `mvn clean install -DskipTests=false` & `mvn clean test-compile failsafe:integration-test` to make sure unit-test and integration-test pass.
- [ ] If this contribution is large, please follow the [Software Donation Guide](https://github.com/apache/incubator-dubbo/wiki/Software-donation-guide).

### CODE REVIEW

**Comment by moyunalibaba** [comment]
LGTM

**Comment by ralf0131** [comment]
What is the purpose of this send pull request?  Can you explain it more?
This looks something like a blog post and should go to https://github.com/apache/incubator-dubbo-website. Moreover, the code should go to https://github.com/apache/incubator-dubbo-samples.

File: dubbo-test/dubbo-test-integration/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+## 具体介绍
+Dubbo-example，是基于PelicanDT实现dubbo环境准备，禁止端口网络访问，执行接口调用验证端口是否禁用示例
+    
+## 前期准备
+1. 本示例程序是基于阿里云ECS或远程Linux服务器完成，只需[购买](https://ecs-buy.aliyun.com/wizard?spm=5176.8789780.1092585.1.520157a8WqaKjA#/prepay/cn-zhangjiakou)阿里云机器，或者选定已准备好的远程服务器即可
```
Comment (ralf0131) [inline dubbo-test/dubbo-test-integration/README.md]: Please remove this. No promotion for Alibaba Cloud.

File: dubbo-test/dubbo-test-integration/src/test/java/org/apache/dubbo/test/DubboLoadBalanceTest.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+import io.restassured.response.Response;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.Test;
+
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+import static io.restassured.RestAssured.given;
+
+/**
```
Comment (ralf0131) [inline dubbo-test/dubbo-test-integration/src/test/java/org/apache/dubbo/test/DubboLoadBalanceTest.java]: Please remove the author name.

File: dubbo-test/dubbo-test-integration/src/test/java/org/apache/dubbo/test/DubboLoadBalanceTest.java [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+ * @author moyun@middleware
+ */
+
+@Slf4j
+public class DubboLoadBalanceTest extends AbstractJUnit4PelicanTests {
```
Comment (ralf0131) [inline dubbo-test/dubbo-test-integration/src/test/java/org/apache/dubbo/test/DubboLoadBalanceTest.java]: Why it is DubboLoadBalanceTest?

File: dubbo-test/dubbo-test-integration/README.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+## 快速入门
+
+### 修改配置
+1. 打开dubbo.properties配置文件，具体路径：Dubbo-example/src/test/resources/env/func/dubbo.properties
+2. 填写ip，userName，password
+
+### 运行示例
+
+本地代码控制远程服务器执行Dubbo验证：
+1. 打开TestDubboNetwork.java，具体路径：Dubbo-example/src/test/java/com/alibaba/pelican/rocketmq/TestDubboNetwork.java
```
Comment (ralf0131) [inline dubbo-test/dubbo-test-integration/README.md]: Why there is rocketmq?

File: dubbo-test/dubbo-test-integration/src/test/resources/dubbo.sh [hunk 5]
```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+cd /root
+yum -y install wget
+yum -y install unzip
+
+if [ ! -f "/root/jdk-8u141-linux-x64.tar.gz" ];then
+    wget http://moyuns.oss-cn-hangzhou.aliyuncs.com/jdk-8u141-linux-x64.tar.gz
```
Comment (ralf0131) [inline dubbo-test/dubbo-test-integration/src/test/resources/dubbo.sh]: It is better to download via official channel, rather that a personal link.

**Comment by beiwei30** [comment]
we cannot accept it. In fact, we have plan to remove dubbo-test from the main project. pls. consider init-ing a new project in github.com/dubbo.

**Comment by moyunalibaba** [comment]
Would you like to consider a project called test-dubbo? I can join in the project!

**Comment by ralf0131** [comment]
Hi, I would suggest to send an email to dev@dubbo.apache.org to discuss it.

**Pull request closed without merging**